### PR TITLE
yngshan/callput_equals

### PR DESCRIPTION
**Pull request by yngshan**

### CODE REVIEW

File: share/contract_categories.yml [hunk 1]
```diff
@@ -11,15 +11,14 @@ callput:
...
-      - CALLE
-      - PUTE
 touchnotouch:
     display_order: 2
     display_name: "Touch/No Touch"
     is_path_dependent: 1
     supported_expiries:
       - intraday
       - daily
+      - tick
```
Comment (junbon) [inline share/contract_categories.yml]: Why this change?

Comment (yngshan) [reply share/contract_categories.yml]: In fact , it is in master 
https://github.com/binary-com/perl-Finance-Contract/blob/master/share/contract_categories.yml#L23

Let me merge the code again

File: share/contract_types.yml [hunk 2]
```diff
@@ -30,9 +30,9 @@ CALL:
...
 CALLE:
   id: 290
-  category: callput
+  category: callputequal
   display_name: Higher or equal
-  other_side_code: PUT
+  other_side_code: PUTE
   payout_type: binary
   payouttime: end
   pricing_code: CALL
```
Comment (shahrizal-shaari) [inline share/contract_types.yml]: Why pricing code is CALL and not CALLE?

Comment (yngshan) [reply share/contract_types.yml]: It using the same pricing code except CALLE has extra commission

File: share/contract_categories.yml [hunk 3]
```diff
@@ -104,6 +102,18 @@ highlowticks:
     available_types:
       - TICKHIGH
       - TICKLOW
+callputequal:
+    display_order: 9
+    display_name: "Rise/Fall Equal"
```
Comment (rakshit-binary) [inline share/contract_categories.yml]: do we need display name or display order ?

Comment (yngshan) [reply share/contract_categories.yml]: The display name of the contract type is higher/lower (on contracts.yml).
